src/classes/HH_HouseholdNaming_TEST.cls

```diff
@@ -109,50 +109,106 @@ public class HH_HouseholdNaming_TEST {
                     npo02__Excluded_Account_Opp_Types__c = null,
                     npo02__Excluded_Contact_Opp_Rectypes__c = null,
-                    npo02__Excluded_Contact_Opp_Types__c = null                    
+                    npo02__Excluded_Contact_Opp_Types__c = null
                 ));
-                
+
             npe01__Contacts_and_Orgs_Settings__c contactSettingsForTests = UTIL_CustomSettingsFacade.getContactsSettingsForTests(new npe01__Contacts_and_Orgs_Settings__c (
                 npe01__Account_Processor__c = CAO_Constants.ONE_TO_ONE_PROCESSOR,
                 npe01__Enable_Opportunity_Contact_Role_Trigger__c = true,
                 npe01__Opportunity_Contact_Role_Default_role__c = 'Donor'
             ));
-        
+
         integer i;
-        
+
         list<Contact> insertlist = new list<Contact>();
         for (i = 0; i < 20; i++){
             Contact c = new Contact(LastName = 'LastName' + i, FirstName = 'FirstName' + i, Salutation = 'Mr.');
             if (i == 1){
                 c.npo02__Naming_Exclusions__c = 'Household Name';
             }
-            
+
             insertlist.add(c);
-        }       
-                
+        }
+
         insert insertlist;
         list<id> householdids = new list<id>();
-        
+
         list<Contact> clist = [select npo02__Naming_Exclusions__c, npo02__Household__r.id, npo02__Household__r.Name, npo02__Household__r.npo02__Informal_Greeting__c, npo02__Household__r.npo02__Formal_Greeting__c from Contact where id IN :insertlist limit 2000];
-        
+
         for (Contact c : clist){
             if (c.npo02__household__r.id != null)
-              householdids.add(c.npo02__household__r.id);          
+              householdids.add(c.npo02__household__r.id);
         }
         system.assert(householdids.size() > 0);
-        
-        //use future to prevent the before HH trigger from firing    
-        test.starttest();         
+
+        //use future to prevent the before HH trigger from firing
+        test.starttest();
         HH_HouseholdNaming.FutureUpdateNames(householdids);
         test.stoptest();
         //test a single member household, lastname, firstname, householdname:
         Contact con = [select npo02__Household__r.Name, npo02__Household__r.npo02__Informal_Greeting__c, npo02__Household__r.npo02__Formal_Greeting__c from Contact where FirstName = 'FirstName0'];
-        system.assertEquals('LastName0 ' +  system.label.npo02.DefaultHouseholdName, con.npo02__household__r.Name);  
+        system.assertEquals('LastName0 ' +  system.label.npo02.DefaultHouseholdName, con.npo02__household__r.Name);
         system.assertEquals('FirstName0',con.npo02__household__r.npo02__Informal_Greeting__c);
         system.assertEquals('Mr. FirstName0 LastName0', con.npo02__household__r.npo02__Formal_Greeting__c);
-        
+
     }
-    
+
+    public static testMethod void HouseholdNamingActionTest(){
```

Comment:
* I suggest to annotate test methods with `@IsTest` for better readability.
* Methods should use the `lowerCamelCase` naming convention. Please rename accordingly.